src/seedsigner/models/psbt_parser.py

```diff
@@ -390,3 +390,13 @@ def verify_multisig_output(self, descriptor: Descriptor, change_num: int) -> boo
         is_owner = descriptor.owns(output)
         # print(f"{self.psbt.tx.vout[i].script_pubkey.address()} | {output.value} | {is_owner}")
         return is_owner
+
+    
+    def get_sum_of_outputs_excluding_change(self):
+        """
+            Returns the sum of all outputs excluding change amount.
```

Comment:
It's implicit in this description, but wouldn't hurt to explicitly note that the fee is excluded from this result.